prep-html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+  <!DOCTYPE html>
+    <html>
+      <body>
+        <h1>My favorite Starwars character is Chewbacca</h1>
+        <h2> <img
+          src="https://lumiere-a.akamaihd.net/v1/images/5e94826f7d7499000120d564-image_f9b9d30e.jpeg?region=0%2C48%2C1536%2C768&width=960"
+          alt="Chewbacca" width="300" height="300"</h2>
```

Comment:
Hi Alex, there's some mixups in your HTML here, the `img` element needs and ending `>` right angle bracket. I recommend removing the `<h2>` tags too.

Normally our code checkers catch this, let's have Cody take a look at your setup after QnA tonight.